app/scripts/modules/core/src/account/account.service.ts

```diff
@@ -76,6 +76,13 @@ export class AccountService {
     });
   }
 
+  public getArtifactAccounts(): ng.IPromise<IAccount[]> {
```

Comment:
Isn't the type of an artifact account very different than an `IAccount`? If so, I would also consider putting it into its own service.

Reply:
Right now it's basically the same - I figured once we make it different I'll pull it into a separate service & interface - WDYT?

Reply:
Sounds good.